Makefile

```diff
@@ -33,5 +33,9 @@ dist/${PACKAGE_NAME}: build-dist ## Build a binary tarball
 
 package: dist/${PACKAGE_NAME}
 
+wheel:
```

Comment:
It's not *terribly* important in this case, but new rules like this should be added to the `.PHONY` pseudo-rule; otherwise if there happens to be a file named `wheel` in the local directory, running `make wheel` will be a no-op

Reply:
I've abandoned this path because I am terrible at makefiles.